Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -139,6 +139,19 @@ static RefPtr<Element> anchorElementForPriorityCandidate(Element* element)
     return nullptr;
 }
 
+static ScrollAnchoringController* scrollAnchoringControllerForElement(Element& element)
```

Comment:
It's confusing that we have this and `RenderObject::findScrollAnchoringControllerForRenderer()` with similar names, yet `findScrollAnchoringControllerForRenderer()` walks the ancestor tree, but this does not. They need more distinct names.

Reply:
Right I think findScrollAnchoringControllerForRenderer is inadequately named. Changed this to searchParentChainForScrollAnchoringController to make it more clear we are walking the ancestor tree.